symbol_factory.py

```diff
@@ -1,9 +1,13 @@
+from helper import my_dict
 class SymbolFactory:
     def get_string(self, single_simbol):
-        """
-          Returns the corresponding symbol's string shape using only |, _, -, /, \,
-          Look at the example output in README.md.
+      return my_dict[single_simbol]
 
-          clue 1: use a dictionary to match the hardcode string with the corresponding symbol.
-        """
-        pass
\ No newline at end of file
+
+        #"""
+          #Returns the corresponding symbol's string shape using only |, _, -, /, \,
+          #Look at the example output in README.md.
+
+          #clue 1: use a dictionary to match the hardcode string with the corresponding symbol.
+        #"""
```

Comment:
Similar than above, this should be the first lines of code of the method `get_string`, they are describing what the `get_string` method is doing.

Reply:
Aquí